hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java

```diff
@@ -248,5 +260,34 @@ public void testECStreamCapability() throws Exception {
       assertFalse(os.hasCapability(StreamCapabilities.HSYNC),
           "ECKeyOutputStream should not support hsync()!");
     }
+    testEncryptedStreamCapabilities(true);
+  }
+
+  private void testEncryptedStreamCapabilities(boolean isEC) throws IOException,
+      GeneralSecurityException {
+    KeyOutputStream kos;
+    if (isEC) {
+      kos = mock(KeyOutputStream.class);
+    } else {
+      kos = mock(ECKeyOutputStream.class);
+    }
```

Comment:
Nit: the name and meaning of `isEC` are inverse (i.e. it tests EC when `isEC=false`).  But the end result is the same, both cases are covered.